partials/_article.scss

```diff
@@ -0,0 +1,101 @@
+
+.articles {
+    width: 100%;
+
+    &__header {
+        text-align: center;
+        font-size: 28px;
+        max-width: 710px;
+        margin: auto;
+        padding: 0px 20px 96px;
+
+        @include media ( ">md" ) {
+            font-size: 32px;
+        }
+    }
+}
+
+.separator {
+    height: 5px;
+    background-color: $tws_coral;
+    width: 50px;
+    align-items: center;
+    margin: auto;
+    margin-bottom: 32px;
+}
+
+.coral {
+    color: $tws_coral;
+}
+
+.gallery {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    margin-bottom: 40px;
+
+    &__tile {
+        max-width: 1264px;
+        width: 100%;
+        text-align: center;
+        align-items: center;
+        display: flex;
+        flex-direction: column;
+        list-style: none;
+        padding: 0;
+        margin: 0;
+
+        @include media ( ">md" ) {
+            width: fit-content;
+            flex-direction: row;
+            margin: auto;
+            padding-left: 30px;
+        }
+
+        &-row {
```

Comment:
I think you slightly confuse the BEM rules. Instead of this structure and classnames, it should rather be something like:
```
 .gallery {
    &__tile {
      ....
     }
     
     &__row {
        ....
     }
     
     &__item {
      ...
     }
     
 }
```
even if an element is a direct child of some parent it doesn't automatically mean it has to contain its classname if that makes sense